pkg/controller/cassandracluster/node_operations_test.go

```diff
@@ -484,7 +484,7 @@ func TestHostIDMap(t *testing.T) {
 			"value": {"10.244.3.20": "ac0b9f2b-1eb4-40ca-bc6e-68b37575f019"},
 			"timestamp": 1528850319,
 			"status": 200}`))
-	jolokiaClient, _ := NewJolokiaClient(host, JolokiaPort, nil,
+	jolokiaClient, _ := NewJolokiaClient(host, cassandraJolokia, nil,
```

Comment:
@allamand Why `CassandraJolokia` ?? Jolokia is used to access JMX even if it's not used with Cassandra

Reply:
the vars was declared in 2 different places, I keep the name of the second.. I can rename it

Reply:
Yeah I would keep `JolokiaPort` instead. It'll also make things simpler for the pending PRs.

Reply:
ok I change it